src/objects.c

```diff
@@ -903,8 +903,6 @@ CK_RV p11prov_obj_from_handle(P11PROV_CTX *ctx, P11PROV_SESSION *session,
     if (obj == NULL) {
         return CKR_HOST_MEMORY;
     }
-    obj->handle = handle;
-    obj->slotid = p11prov_session_slotid(session);
```

Comment:
Please explain this change.

Reply:
In `p11prov_obj_new` `handle` and `slotid` already init with other `P11PROV_OBJ` fields. It seems logical that initialization takes place there

Reply:
Ok the reason I asked is that this is an extraneous change to the rest of the PR, but it is isolated to a separate commit, so it is ok.